scripts/generate_prefrosh.py

```diff
@@ -0,0 +1,18 @@
+import sqlalchemy
```

Comment:
Also either in this file or on the wiki, you should write documentation on how to set up for and run this script.